src/stylesheets/app.scss

```diff
@@ -0,0 +1,276 @@
+@import "base/typography";
+@import "base/vars";
+
+body{
+  margin: 0;
+  background-color: #d2e7e2; // only temporary
+}
+ul{
+  padding: 0;
+}
+sup{
+  color: #e95e67;
+  text-decoration: underline;
+}
+
+/*** Top Fold styles ***/
+#top-fold{
+  background: #d2e7e2;
+  padding-top: 50px;
+  overflow: hidden;
+  padding-bottom: 15%; //only temporary
+
+  #title-box{
+    position: relative;
+  }
+  #title{
+    margin: auto;
+    max-width: 50%;
+    @media (max-width: 600px){
+      max-width: 85%;
+      margin-bottom: 8%;
+    }
+  }
+
+  #subtitle{
+    color: #0e5254;
+    font-family: 'Myriad Pro';
+    font-size: 14px;
+    letter-spacing: 3px;
+    margin-bottom: 2%;
+    text-align: center;
+    text-transform: uppercase;
+  }
+
+  #yellow-pill{
+    left: 22%;
+    top: -30%;
+    width: 7%;
+    position: absolute;
+    transform: rotate(
+      35deg
+    );
+    @media (max-width: 600px){
+      width: 20%;
+      top: -55%;
+      left: 1%;
+      transform: rotate(
+        40deg
+      );
+    }
+  }
+
+  #pills{
+    left: 12%;
+    position: absolute;
+    top: 65%;
+    width: 10%;
+    @media (max-width: 600px){
+      display: none;
+    }
+  }
+
+  #more-pills{
+    position: absolute;
+    top: -20%;
+    right: 12%;
+    width: 20%;
+    transform: rotate(10deg);
+    @media (max-width: 600px){
+      width: 60%;
+      top: -40%;
+      right: -35%;
+    }
+  }
+
+  #purple-pills{
+    position: absolute;
+    right: 10%;
+    width: 12%;
+    @media (max-width: 600px){
```

Comment:
since you use `max-width: 600px` a lot, might be good to save that to a variable

Reply:
you also may find yourself needing more breakpoint sized variables than just this one. some generic, yet good ones i find myself using a lot are:
```
400px
768px
1000px
1200px
1400px
```
but do what feels right for the project